testing/php7-rrd/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Pascal Hofmann <mail@pascalhofmann.de>
+# Contributor: Micka�l Remars <github@remars.com>
+# Maintainer: Pascal Hofmann <mail@pascalhofmann.de>
+pkgname=php7-rrd
+_pkgreal=rrd
+pkgver=2.0.1
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
```

Comment:
Why not something like this?
```
_phpver=7
_pkgreal=rrd
pkgname="php${_phpver}-${_pkgreal}"
```

Reply:
This was present in the original packaging from Pascal Hofmann. I guess he made this to make the build compatible with any version of php.